docs/RNCamera.md

```diff
@@ -598,6 +599,8 @@ writeExif = {
 
 - `orientation` (string or number). Specifies the orientation that us used for taking the picture. Possible values: `"portrait"`, `"portraitUpsideDown"`, `"landscapeLeft"` or `"landscapeRight"`.
 
+- `path` (file path on disk). Specifies the path on disk to save picture to.
```

Comment:
What about warning that setting this, can slow down the proccess of taking a picture?

Reply:
Without path option, it saves in Android/data/{app_package_name}/files so I think it won't much affect the speed if picture is save in another location.

Reply:
@MateusAndrade @sibelius 
Sorry for confusion but my above test was an fault might be due to setting debugger on. Now, I think it won't affect the speed as in both cases we have to write file into directory.